### PR TITLE
README: fix `init-wallet` cmd

### PR DESCRIPTION
**Pull request by erikarvstedt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,11 +52,10 @@ file. This will take a few seconds in any case.
 
 ```shell
 $ lndinit -v init-wallet \
-    --secret-source=file \
```
Comment (guggero) [inline README.md:55]: I'd like to keep the examples as explicit as possible, so please leave this in, even though it's the default.

File: README.md [hunk 2]
```diff
@@ -52,11 +52,10 @@ file. This will take a few seconds in any case.
...
     --file.seed=/safe/location/seed.txt \
     --file.wallet-password=/safe/location/walletpassword.txt \
-    --output-wallet-dir=$HOME/.lnd/data/chain/bitcoin/mainnet \
-    --validate-password
+    --init-file.output-wallet-dir=$HOME/.lnd/data/chain/bitcoin/mainnet \
```
Comment (guggero) [inline README.md:57]: Nice catch!

**Review by guggero — CHANGES_REQUESTED** [review]
Thanks for the fix!

**Review by guggero — APPROVED** [review]
LGTM :tada:

**Review by thinkmassive — APPROVED** [review]
nice, thank you @erikarvstedt!

**Pull request merged**